app/views/projects/edit.html.erb

```diff
@@ -1,22 +1,43 @@
 <div class="p-4">
   <div class="grid gap-4 xl:grid-cols-2 2xl:grid-cols-3">
     <!--Edit general details form -->
-    <div class="p-4 bg-white border border-gray-200 rounded-lg shadow-sm 2xl:col-span-2 dark:border-gray-700 sm:p-6 dark:bg-gray-800" data-controller="slugify">
-      <h2 class="text-xl font-semibold leading-tight text-gray-900 dark:text-gray-100">
-        <%= t(:'.general.title') %>
+    <div
+      class="
+        p-4
+        bg-white
+        border
+        border-gray-200
+        rounded-lg
+        shadow-sm
+        2xl:col-span-2
+        dark:border-gray-700
+        sm:p-6
+        dark:bg-gray-800
+      "
+      data-controller="slugify"
+    >
+      <h2
+        class="
+          text-xl
+          font-semibold
+          leading-tight
+          text-gray-900
+          dark:text-gray-100
+        "
+      >
+        <%= t(:".general.title") %>
       </h2>
       <%= form_with(model: @project, url: namespace_project_path, method: :patch) do |form| %>
-        <%= turbo_frame_tag "project_form" do %>
```

Comment:
Do we no longer require the `turbo_frame_tag` here?

Reply:
Yeah, this form never actually used a turbo frame, I just was copying code.  We need to break this one up anyways because of the namespace stuff so this will change again shortly.